source/_integrations/plugwise.markdown

```
@@ -30,40 +30,42 @@ ha_platforms:
-Platforms available - depending on your Smile and setup include:
-
- - `climate` (for the stand-alone Anna, for Adam, a climate entity is shown for each zone containing devices like an Anna or another type of wired-thermostat, Jip or Lisa combined with one or more Tom/Floor devices)
- - `binary_sensor` (for showing the status of e.g. domestic hot water heating or secondary heater)
- - `button` (for the Adam and the non-legacy Anna and P1 gateways)
- - `number` (for changing a boiler setpoint, a temperature offset)
- - `sensor` (for all relevant products including the Smile P1)
- - `select` (for changing a thermostat schedule, a regulation mode (Adam only))
- - `switch` (for Plugs connected to Adam, or Circles and Stealths connected to a Stretch)
+This integration supports one or multiple Plugwise Gateways available on your network. You can connect to these gateways using your browser, the Plugwise App, or Home Assistant. There are 4 [supported](#supported-devices) types of gateways:
 
+- Full zonecontrol using the [Adam](https://www.plugwise.com/en_US/zonecontrol) using [additional devices](#devices-overview) such as smart valves and smart-plugs.
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)